### PR TITLE
Enhance/payment page dorpdown and back button

### PR DESCRIPTION
**Pull request by NAbdulla1**
### All Submissions:

* [x] My code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)
* [x] My code satisfies feature requirements
* [x] My code is tested
* [x] My code passes the PHPCS tests
* [x] My code has proper inline documentation
* [ ] I've included developer documentation (optional)
* [x] I've added proper labels to this pull request

### Changes proposed in this Pull Request:
Enhance UI of payment page
add back link from payment settings page.

image

image

Closes [Dokan pro issue 1860](https://github.com/weDevsOfficial/dokan-pro/issues/1860) and #1609 
### How to test the changes in this Pull Request:

1. Go to Vendor Dashboard > settings > payment
2. see the improved dropdown
3. click on a method from dorpdown
4. see 'Back' link at the top left corner, 

### Changelog entry

> **enhance:** improved UI of Payment settings page
> **fix:** changing dokan vendor dashboard page slug gives 404 error

### PR Self Review Checklist:

* Code is not following code style guidelines
* Bad naming: make sure you would understand your code if you read it a few months from now.
* KISS: Keep it simple, Sweetie (not stupid!).
* DRY: Don't Repeat Yourself.
* Code that is not readable: too many nested 'if's are a bad sign.
* Performance issues
* Complicated constructions that need refactoring or comments: code should almost always be self-explanatory.
* Grammar errors.

### CODE REVIEW

File: templates/settings/payment.php [hunk 1]
```diff
@@ -14,33 +14,35 @@
...
+        <div>
+            <div id="vendor-dashboard-payment-settings-toggle-dropdown">
+                <a id="toggle-vendor-payment-method-drop-down"> <?php esc_html_e( 'Add Payment Method', 'dokan-lite' ); ?></a>
+                <div id="vendor-payment-method-drop-down-wrapper">
+                    <div id="vendor-payment-method-drop-down">
+                        <?php if ( is_array( $unused_methods ) && ! empty( $unused_methods ) ) : ?>
+                            <ul>
+                            <?php foreach ( $unused_methods as $method_key => $method ) :?>
+                                <li>
+                                    <a href="<?php echo esc_url( home_url( "dashboard/settings/payment/manage-" . $method_key ) ); ?>">
```
Comment (tanmayjay) [inline templates/settings/payment.php:26]: use `dokan_get_page_url`

**Review by tanmayjay — CHANGES_REQUESTED** [review]

**Comment by Rat01047** [comment]
Issue-1: 
Payment Settings Page Design Break

Scenario: Vendor Dashboard > Payment Settings Page > Design Break + Text not showing properly

Screenshot for reference:
image

**Comment by Rat01047** [comment]
Issue-2: 
Payment Setup page Design break on Bank Transfer for a new Vendor
 
Scenario: Create New Vendor > Payment Setup page > Design break for Bank Transfer
 

Screenshot for reference:
image

**Comment by Rat01047** [comment]
Issue-3:
Payment Setup page Design break on Bank Transfer for a new Vendor

Scenario: Create New Vendor > Payment Setup page > Design break for Bank Transfer


Screenshot given for reference:
image

**Review by nurul-umbhiya — APPROVED** [review]

**Pull request merged**